epub34/authoring/index.html

```diff
@@ -221,12 +221,27 @@ <h4>Relationship to HTML</h4>
 						the various changes to HTML and the technologies it references to ensure they keep their
 						processes up to date.</p>
 
+					<p>The [[html]] standard defines a single content model with rules for expressing a tag set using
+						either the <a data-cite="html#syntax">HTML syntax</a> (what has always been considered HTML) or
+						the <a data-cite="html#the-xhtml-syntax">XML syntax</a> (what was formerly known as XHTML). EPUB
+						3 allows authoring of content documents using either syntax even though the [[html]] standard <a
+							data-cite="html/xhtml.html#the-xhtml-syntax">no longer recommends the use of the XML
+							syntax</a>. The Working Group recognizes that XML remains an integral technology in the
+						publishing ecosystem and will not remove support for the XML syntax from EPUB 3. Regardless,
+						publishers that prefer to keep using the XML syntax will need to monitor future support for it,
+						and may have to adapt to the HTML syntax to gain access to some features of [[html]].</p>
+
+					<div class="issue" data-number="2715" title="Adoption of the HTML syntax">
+						<p>The change to allow both syntaxes of HTML is an open issue in EPUB 3.4. The above paragraph
+							was added to help explain the change, but would be reverted if the HTML syntax is not
```

Comment:
```suggestion
							was added to help explain the change, but would be amended if the HTML syntax is not
```

(Not sure that we would just revert to the original text. I believe the emphasis on the fact that XHTML is 'just' the XML syntax to the moving target called "html" is important.)